source/features/conventional-commits.tsx

```diff
@@ -7,65 +7,42 @@ import observe from '../helpers/selector-observer.js';
 import {commitTitleInLists} from '../github-helpers/selectors.js';
 import {parseConventionalCommit, removeCommitAndScope} from '../helpers/conventional-commits.js';
 
-const types = new Map([
-	['feat', 'Feature'],
-	['fix', 'Fix'],
-	['chore', 'Chore'],
-	['docs', 'Docs'],
-	['build', 'Build'],
-	['refactor', 'Refactor'],
-	['test', 'Test'],
-	['ci', 'CI'],
-	['perf', 'Performance'],
-]);
-
-function createLabelElement(type: string, scope?: string): JSX.Element {
-	const label = types.get(type)!;
-
-	return (
-		<span className={`IssueLabel hx_IssueLabel rgh-commit-type-label rgh-commit-type-label-${type}-colors`}>
-			{scope ? `${label}: ${scope}` : label}
-		</span>
-	);
-}
-
 function renderLabelInCommitTitle(commitTitleElement: HTMLElement): void {
-	const match = parseConventionalCommit(commitTitleElement.textContent);
+	const textNode = commitTitleElement.firstChild!;
+	const commit = parseConventionalCommit(textNode.textContent);
 
-	const {type, scope} = match?.groups ?? {};
-	if (!type || !types.has(type)) {
+	if (!commit) {
 		return;
 	}
 
-	commitTitleElement.prepend(createLabelElement(type, scope));
-	removeCommitAndScope(commitTitleElement, match!);
-}
+	commitTitleElement.prepend(
+		<span className="IssueLabel hx_IssueLabel mr-2" rgh-conventional-commits={commit.rawType}>
+			{commit.type}
+		</span>,
+		commit.scope ? <em>{commit.scope}</em> : '',
```

Comment:
Badges shouldn't really contain that much information. This matches the size to GitHub's own usage of labels/bubbles.



<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td>
<img width="401" alt="Screenshot 3" src="https://github.com/user-attachments/assets/ad54d9b5-5af5-4bb0-97d4-b88ba82915d0">
	<td>
<img width="401" alt="Screenshot 4" src="https://github.com/user-attachments/assets/730c80af-f50b-4f60-92f3-66ba59b343ea">

</table>

Reply:
This formatting makes it look like the meta information "scope" belongs to the commit message, which is not the case. The scope is meta information for the commit message that is followed by the colon. If at all, it should be visually and semantically grouped with the commit type. The length you see in the commits is specific to the repository you have screenshotted. Scopes usually are not very long as shown in the conventional commit page: 
![image](https://github.com/user-attachments/assets/63195004-5394-4a4e-8f38-e812ecfa189d)

Reply:
Are you sure? Have you looked at a changelog? This is exactly what a conventional changelog looks like, I already mentioned this before.

Reply:
Yes, I am. You can check many other conventional commit reports, and you'll see them using short scopes.

Reply:
I’m not talking about the length but about the scope usage/display:

![IMG_5675](https://github.com/user-attachments/assets/27aed4d8-c3dc-4817-8c76-17c349d8cb32)

Reply:
The scope of a commit is _meta_ information. If you were to have a repository with conventional commits and had the task to revert using conventional commits, you'd get rid of everything before the colon. The actual information is the commit message after the colon.